### PR TITLE
docs: added the "delete" subsection in the configuration.md file

### PR DESCRIPTION
**Pull request by mssola**
Signed-off-by: Miquel Sabaté <msabate@suse.com>

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -344,10 +348,19 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
...
 You must configure one backend; if you configure more, the registry returns an error.
 
-If you are deploying a registry on Windows, be aware that a Windows volume mounted from the host is not recommended. Instead, you can use a S3, or Azure, backing data-store. If you do use a Windows volume, you must ensure that the `PATH` to the mount point is within Windows' `MAX_PATH` limits (typically 255 characters). Failure to do so can result in the following error message: 
+If you are deploying a registry on Windows, be aware that a Windows volume mounted from the host is not recommended. Instead, you can use a S3, or Azure, backing data-store. If you do use a Windows volume, you must ensure that the `PATH` to the mount point is within Windows' `MAX_PATH` limits (typically 255 characters). Failure to do so can result in the following error message:
 
     mkdir /XXX protocol error and your registry will not function properly.
 
+### delete
+
+Use the `delete` subsection to enable the deletion of image blobs. It defaults
```
Comment (RichardScothern) [inline docs/configuration.md]: "image blobs" -> "image blobs and manifests by digest"

**Comment by RichardScothern** [comment]
Thanks you @mssola , one small nit.

**Comment by mssola** [comment]
@RichardScothern done :)

**Comment by RichardScothern** [comment]
:cake:  LGTM

**Pull request merged**